recipes-nodes/render-config/fetch-config.sh

```diff
@@ -22,8 +20,8 @@ case "$1" in
     echo "Fetching configuration..." > /var/volatile/system-api.fifo
 
     (umask 0177 && touch "${INIT_CONFIG_FILE}")
-    curl -fsSL --proxy http://localhost:7937 --retry 3 --retry-delay 60 --retry-connrefused \
-      -H "Metadata: true" -o "${INIT_CONFIG_FILE}" "${INIT_CONFIG_URL}"
+    curl -fsSL --retry 3 --retry-delay 60 --retry-connrefused \
+      -o "${INIT_CONFIG_FILE}" http://localhost:7937/api/l1-builder/v1/configuration
```

Comment:
if you haven't set the env variables correctly during build then it will default to https://hub-atls.builder.flashbots.net
See [here](https://github.com/flashbots/meta-evm/blob/main/recipes-nodes/render-config/render-config.bb#L27)

Reply:
I think there's no need to pass `INIT_CONFIG_URL` since it's always supposed to be behind the proxy right?

Reply:
see my comment above